registry/repository.go

```diff
@@ -89,6 +90,11 @@ type ReferenceTagger interface {
 	TagReference(ctx context.Context, src, dst string) error
 }
 
+// ReferrerFinder provides the Referrers API.
```

Comment:
It would be better if a reference link is provided.

Reply:
Added the link.